package.json

```diff
@@ -53,15 +53,14 @@
     "grunt-coveralls": "^1.0.1",
```

Comment:
need to version bump package.json

Reply:
@austincunningham bumped the version in this case and added CHANGELOG file.

Reply:
@camilamacedo86 @austincunningham Are we planning to release this and update in fh-supercore and fh-mbaas, and release new versions of those then? It doesn't seem worthwhile, just because the minimum version of Node.js was bumped in `engines.node`, since we're already using Node.js 6 in those components, and that's the only user-facing change.

Reply:
@grdryn ,

Is the update automatic? IMHO should we bump the version, however, is not required at all update the fh-supercore and fh-mbaas because of it.

Reply:
The update is not automatic. I like your idea of adding it to UNRELEASED in the changelog :+1:

Reply:
OKs. Let's move forward with the UNRELEASED so.